CHANGELOG.md

```diff
@@ -78,6 +78,8 @@
 * [ENHANCEMENT] Execute health checks in ring client pool concurrently. #237
 * [ENHANCEMENT] Cache: Add the ability to use a custom memory allocator for cache results. #249
 * [ENHANCEMENT] Add the metrics package to support per tenant metric aggregation. #258
+* [ENHANCEMENT] Cache: Add Delete method to cache.Cache interface. #255
+* [ENHANCEMENT] Cache: Add backward compatible metrics exposition for cache. #260
```

Comment:
Clarify the impact of this change please.

Reply:
Changelog is updated with list of metrics with new format.